pdns/dnsdistdist/configure.ac

```diff
@@ -69,6 +69,17 @@ AM_CONDITIONAL([HAVE_CDB], [false])
 
 PDNS_CHECK_LIBCRYPTO
 
+DNSDIST_ENABLE_TLS_PROVIDERS
+
+AS_IF([test "x$enable_tls_providers" != "xno"], [
+  AS_IF([test "x$HAVE_LIBSSL_3_PLUS" != "x1"], [
+    AC_MSG_ERROR([TLS providers support requires OpenSSL >= 3.0])
+  ])
+
+  PDNS_WITH_LIBSSL
```

Comment:
Do we really need to call `PDNS_WITH_LIBSSL` here?

Reply:
It sets up all the necessary paths for openssl so I would say yes it's required.

Reply:
Isn't it getting called on line 76 already?

Reply:
Did you perhaps mean line 72?
`DNSDIST_ENABLE_TLS_PROVIDERS` only checks the openssl version and doesn't concern itself with paths.

Reply:
That's 76 _before_ the PR, sorry, so 87 now.

Reply:
Right.. and since provider support isn't very useful without TLS in our case, we can skip the call here.

Reply:
I'll make the changes, shall I squash after that?

Reply:
Yes, please!